### DISCUSSION TITLE
Chrome_ChildIOT detected without a path or anything else to identify it

### DISCUSSION THREAD

**Discussion by zerthsky** [evilsocket/opensnitch#850] [Q&A] [answered]
Mint 21.1, OpenSnitch 1.6.0.rc3. I know the current version is rc4, but rc3 is in the repo and I don't see anything relevant in the changelog.

The issue may be hard to reproduce so I'm looking mostly for general guidance here. The problem was not present back when I was using Chromium but appeared immediately after I switched to Edge. Chromium is still installed on my system, though.

OpenSnitch reports an outbound attempt by Chrome_ChildIOT. No path shows regardless of what I click in the dialog. If left unattended, it creates a rule as expected. Screenshot attached.

I'm mostly curious about why it has no path. For now I'm not even sure if it is a legitimate Chromium or Edge process at all.

![Screenshot from 2023-02-14 16-28-34](https://user-images.githubusercontent.com/120786124/218753885-570898b1-62e0-4ffe-b929-4120a7afe908.png)

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
Hi @zerthsky ,

My recommendation would be to upgrade v1.6.0rc4, there're 3 important bugs that fix errors adding rules (_Fixed adding interception rules on new and old kernels ( https://github.com/evilsocket/opensnitch/commit/ced9a249338a33f131bb1b72f747135985ab8608 , https://github.com/evilsocket/opensnitch/commit/d31c4e86bf77319d6a9516e479e3cee27d19db60, https://github.com/evilsocket/opensnitch/issues/781)_)

This scenario is likely caused because we failed getting the path of the process, and instead of dropping the package and let chrome retry the connection again, we used /proc/$pid/comm value to associate it the connection.

There're several reasons that can cause this behaviour, do you usually suspend/hibernate your PC?

On the other hand, set LogLevel to DEBUG under Preferences -> Nodes, and when you this problem again, copy right away /var/log/opensnitchd.log to /tmp (for example), and post the log.

Thank you!

**Reply by zerthsky** [reply]
Thank you very much for your prompt response. I set the log level to debug. Will post when I get anything happening.

Not using suspend or hibernate, by the way. It's a desktop and always plugged in, hence I see no point.

Forgot to mention that my Edge is flatpak. However, Chromium is flatpak also and it never gives me this.

**Reply by zerthsky** [reply]
The issue has not happened once in the 3 weeks passed, so I guess it was related to a specific Edge version and no longer relevant. Thank you again for your help.

I'm very happy with OpenSnitch as a whole. It's both educative and fun.

**Comment by ninmba** [comment]
Hello,
I have the same problem when I used Firefox, same process name called Chrome_ChildIOT with no path.
I wasn't using Chrome or Chromium or any other browser based on it, and when I blocked that process, Firefox got blocked and I couldn't establish a connection until I removed the rule.
Debian testing x64 kde, opensnitch v1.6.6

**Reply by gustavo-iniguez-goya** [reply]
hey @ninmba ,

That sounds strange. Do you use Electron apps, like Spotify? Electron based apps and some others use chrome to draw the GUI.
If you're using ProcMonitorMethod 'ebpf' (Preferences->Nodes), and ebpf is working fine, it should have detected correctly the process.
